torchvision/prototype/transforms/functional/_deprecated.py

```diff
@@ -50,3 +52,12 @@ def to_tensor(inpt: Any) -> torch.Tensor:
         "Instead, please use `to_image_tensor(...)` followed by `convert_image_dtype(...)`."
     )
     return _F.to_tensor(inpt)
+
+
+def get_image_size(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> List[int]:
+    warnings.warn(
+        "The function `get_image_size(...)` is deprecated and will be removed in a future release. "
+        "Instead, please use `get_spatial_size(...)` which returns `(h, w)` instead of `(w, h)`."
+    )
+    _, h, w = get_chw(image)
+    return [w, h]
```

Comment:
Moving the method and fixing the bug. Previously it returned (h, w)

Reply:
The other deprecated functions here emit a warning, but otherwise use the old code. Should we also do that here?